test/simplifyExpression/arithmeticSearch/arithmeticSearch.test.js

```diff
@@ -6,11 +6,11 @@ function testArithmeticSearch(exprStr, outputStr) {
   TestUtil.testSimplification(arithmeticSearch, exprStr, outputStr);
```

Comment:
This still uses `mathjs.parse`, see
https://github.com/socraticorg/mathsteps/blob/master/test/TestUtil.js#L26-L33